pkg/api/grpcServer.go

```diff
@@ -71,11 +94,8 @@ func NewGRPCServer(aspectDispatcher adapterManager.AspectDispatcher, tracer trac
 }
 
 // dispatcher does all the nitty-gritty details of handling the mixer's low-level API
-// protocol and dispatching to the right API handler.
-func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string,
-	getState func() (request proto.Message, response proto.Message, requestAttrs *mixerpb.Attributes, responseAttrs *mixerpb.Attributes, result *rpc.Status),
-	worker func(ctx context.Context, requestBag *attribute.MutableBag, responseBag *attribute.MutableBag,
-		request proto.Message, response proto.Message)) error {
+// protocol and dispatching to the right API dispatchWrapperFn.
+func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string, getState stateGetterFn, worker dispatchFn) error {
```

Comment:
Addressed (1).  I've left (2) alone for now. I think there is work needed to refactor how config gets resolved and passed around as part of this dispatch() call. If OK with you, I'd like to hold off on that bit of clean-up until tackling the larger task in a more wholesale manner.